app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -14,9 +14,17 @@
   header_classes = %w[gem-c-layout-header govuk-header]
   header_classes << "gem-c-layout-header--#{environment}" if environment
   header_classes << "gem-c-layout-header--no-bottom-border" if remove_bottom_border
+
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
```

Comment:
Can the lines above for `header_classes` be removed now?

Reply:
Whoops, good spot, have removed.